### ISSUE TITLE
Crash with 2.0.2

### ISSUE THREAD

**Issue by katubug** [bigenergy/glassential-renewed#3]
I saw you just closed out a ticket and you've got 3 mod updates already today, so I really hate to report this, but I'm getting a crash with the latest version of the mod. Reverting to 1.4.0 fixes it. My forge version is 47.2.19

Here's my crash report and latest.log: https://gist.github.com/katubug/ad548d8c063277fc78bb14da730b926a

**Comment by bigenergy** [comment]
It doesn't look like a crash in my mod, there are a lot of broken mods in your log.
Write in more detail what you did

**Comment by katubug** [comment]
All I did was update Glassential to 2.0.2 and attempt to start the game.

I did a little extra testing just now, and it seems that it's a conflict with the mod Copperative. Both mods work fine separately, but not together.

Here's the updated log and crash report in a smaller instance: https://gist.github.com/katubug/58ae69124dd3226abdacc70c900deebf

And here's the issue reported to Copperative: https://github.com/Xaidee/coopperative/issues/43
